.circleci/config.yml

```diff
@@ -129,7 +129,7 @@ commands:
           when: always
       - run:
           name: Run ffi cucumber scenarios
-          command: cd integration_tests && mkdir -p cucumber_output && node_modules/.bin/cucumber-js --tags "not @long-running and not @broken and not @flaky and @wallet-ffi" --format json:cucumber_output/tests_ffi.cucumber --exit
+          command: cd integration_tests && mkdir -p cucumber_output && touch cucumber_output/tests_ffi.cucumber && node_modules/.bin/cucumber-js --tags "not @long-running and not @broken and not @flaky and @wallet-ffi" --format json:cucumber_output/tests_ffi.cucumber --exit
```

Comment:
```suggestion
          command: cd integration_tests && mkdir -p cucumber_output && touch cucumber_output/tests_ffi.cucumber && node_modules/.bin/cucumber-js --tags "not @long-running and not @broken and not @flaky and @wallet-ffi" --format json:cucumber_output/tests_ffi.cucumber --exit
          when: always
```

Reply:
This runs the FFI test even if the previous step failed

Reply:
Thanks.